build-tools/packages/build-cli/src/commands/merge/pullRequest.ts

```diff
@@ -0,0 +1,232 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Flags } from "@oclif/core";
+import { BaseCommand } from "../../base";
+import {
+	Repository,
+	createComment,
+	getPullRequestInfo,
+	listCommitsPullRequest,
+	mergePullRequest,
+} from "../../lib";
+import { CommandLogger } from "../../logging";
+
+interface CommitData {
```

Comment:
nit: naming convention
```suggestion
interface ICommitData {
```